lib/rmt/cli/export.rb

```diff
@@ -0,0 +1,25 @@
+# rubocop:disable Rails/Output
+
+class RMT::CLI::Export < RMT::CLI::Base
+
+  desc 'settings PATH', 'Store repository settings at given path'
+  def settings(path)
+    File.write(File.join(path, 'repos.json'), Repository.only_mirrored.pluck(:id).to_json)
```

Comment:
Hmm, yeah, for custom repos we'd need a bit more than just the ids.

Reply:
Well, but then we should have worked on the "custom repos" card first to define how their information is stored. For the current feature set of RMT, this repos.json format it all it needs. Let's extend it when we get there. I don't want to start another card before finally finishing this one.

Reply:
Yep! Just saying that the format is likely to change soon, which might be an issue if SUMA would want to use it.

Reply:
Agreed, I would suggest to not call this feature "done done" in the sense of "being ready to be picked up by SUMA". Let's do that when we have the custom repos feature defined and implemented. Okay with you, @kalabiyau ?